### PR TITLE
Add a step to decouple the conf and data folders from the binaries

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1457.surge.sh

Updated pages:
- [Linux executable (.tar)](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/linux/tarball)
- [Windows installation](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/windows)

File: modules/ROOT/pages/installation/linux/tarball.adoc [hunk 1]
```diff
@@ -53,6 +53,8 @@ See the link:https://neo4j.com/terms/licensing/[Neo4j licensing] page for detail
 +
 ** Set the environment variable `NEO4J_ACCEPT_LICENSE_AGREEMENT=eval`.
 ** Run `<NEO4J_HOME>/bin/neo4j-admin server license --accept-evaluation`.
+. (Optional) Decouple the data and configuration directories from the binary files by setting the environment variable `NEO4J_CONF` and `dbms.directories.data` to point to the desired locations.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/installation/linux/tarball.adoc:56]:
```suggestion
. (Optional) Decouple the data and configuration directories from the binary files by setting the environment variable `NEO4J_CONF` and `server.directories.data` to point to the desired locations.
```

File: modules/ROOT/pages/installation/windows.adoc [hunk 2]
```diff
@@ -30,6 +30,8 @@ See the link:https://neo4j.com/terms/licensing/[Neo4j licensing] page for detail
 +
 ** Set it as an environment variable using `set NEO4J_ACCEPT_LICENSE_AGREEMENT=eval`.
 ** Run `<NEO4J_HOME>\bin\neo4j-admin server license --accept-evaluation`.
+. (Optional) Decouple the data and configuration directories from the binary files by setting the environment variable `NEO4J_CONF` and `dbms.directories.data` to point to the desired locations.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/installation/windows.adoc:33]:
```suggestion
. (Optional) Decouple the data and configuration directories from the binary files by setting the environment variable `NEO4J_CONF` and `server.directories.data` to point to the desired locations.
```

**Comment by NataliaIvakina** [comment]
Looks good, @renetapopova! I have a question about the macOs installation section: Do we need to add this step there?

File: modules/ROOT/pages/installation/linux/tarball.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/installation/linux/tarball.adoc:56]: Good catch!! Thanks, @NataliaIvakina!

**Comment by renetapopova** [comment]
> Looks good, @renetapopova! I have a question about the macOs installation section: Do we need to add this step there?

Hmm, that's a good point. I'll add it there as well.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1457.surge.sh

Updated pages:
- [Linux executable (.tar)](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/linux/tarball)
- [Windows installation](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/windows)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1457.surge.sh

Updated pages:
- [Linux executable (.tar)](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/linux/tarball)
- [Windows installation](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/windows)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1457.surge.sh

Updated pages:
- [Linux executable (.tar)](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/linux/tarball)
- [macOS installation](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/osx)
- [Windows installation](https://neo4j-docs-operations-1457.surge.sh/operations-manual/5/installation/windows)

**Pull request merged**